gobblin-audit/src/main/java/gobblin/audit/values/HasAlias.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.audit.values;
+
+import com.typesafe.config.Config;
+
+/**
+ * An interface to allow classes to have aliases.
+ * An alias is meant to be used in job {@link Config} instead of the complete cannonical name of the class for instantiation
+ *
+ * @see {@link AliasUtils#createWithAlias(Class, String, Object...)}
+ */
+public interface HasAlias {
```

Comment:
Can you make this a class annotation instead? Checks for class alias names can be checked statically, without having to instantiate objects.